src/scenes/Partners/Detail/PartnerDetail.tsx

```diff
@@ -60,21 +109,97 @@ export const PartnerDetail = () => {
               >
                 <PencilCircledIcon />
               </IconButton>
-            ) : null} */}
+            </Tooltip>
+          </header>
+          <div className={classes.subheader}>
+            <Typography variant="h4">
+              {partner ? 'Partner Information' : <Skeleton width={200} />}
+            </Typography>
+            {partner && (
+              <Typography variant="body2" color="textSecondary">
+                Created {formatDate(partner.createdAt)}
+              </Typography>
+            )}
+          </div>
+          <Grid container spacing={1} alignItems="center">
+            <Grid item>
+              <DataButton
+                onClick={() => editPartner('pmcEntityCode')}
+                secured={partner?.pmcEntityCode}
+                redacted="You do not have permission to view PMC Entity Code"
+                children={`PMC Entity Code${
+                  partner?.pmcEntityCode.value
+                    ? `: ${partner.pmcEntityCode.value}`
+                    : ''
+                }`}
+              />
+            </Grid>
+            <Grid item>
+              <DataButton
+                onClick={() => editPartner('active')}
+                secured={partner?.active}
+                redacted="You do not have permission to view Status"
+                children={partner?.active.value ? 'Active' : 'Inactive'}
+              />
+            </Grid>
+            <Grid item>
+              <BooleanProperty
+                label="Global Innovations Client"
+                redacted="You do not have permission to view whether this is a Global Innovations Client"
+                data={partner?.globalInnovationsClient}
+                wrap={(node) => <Grid item>{node}</Grid>}
+              />
+            </Grid>
+          </Grid>
+          <Grid container spacing={3}>
+            <Grid item xs={6} className={classes.cardSection}>
+              <Typography variant="h3">Partner Types</Typography>
+              <PartnerTypeCard
+                partner={partner}
+                onEdit={() => editPartner(['types', 'financialReportingTypes'])}
+                className={classes.card}
+              />
+            </Grid>
+            <Grid item xs={6} className={classes.cardSection}>
+              <Typography variant="h3">Address</Typography>
+              <AddressCard
+                partner={partner}
+                onEdit={() => editPartner('address')}
+                className={classes.card}
+              />
+            </Grid>
+          </Grid>
+          <div className={classes.sectionTitle}>
+            <Typography variant="h3">Point of Contact</Typography>
+            <IconButton
+              color="primary"
+              aria-label="edit partner"
+              onClick={() => editPartner('pointOfContactId')}
+            >
+              <AddCircle />
+            </IconButton>
+            <Typography>Edit</Typography>
```

Comment:
This seems like another place where we should maybe use the same FAB as above. I realize this reflects the design, so maybe we need to ask the design team for clarification on this.

If they want to stick with the existing design, we should make sure the word "Edit" is clickable

Reply:
Yeah we've translated the word to a tooltip so we don't have to make the word clickable with the FAB

Reply:
Sorry guys not sure what the decision here is.  Keep as is (edit word non-clickable) or change it to a tooltip, something else?

Reply:
Let's not match the designs and instead make it a Fab with a tooltip. This makes it consistent with the rest of the app.

If design doesn't like this then we can talk about the reasons for us doing this (text is clickable that isn't styled as it) and possibly update across the app.